libs/MVS/Scene.cpp

```diff
@@ -1373,4 +1373,97 @@ bool Scene::ScaleImages(unsigned nMaxResolution, REAL scale, const String& folde
 	}
 	return true;
 } // ScaleImages
+
```

Comment:
move /*----------------------------------------------------------------*/ up, next to the end of the previous function (like in all the other palces)